BUILDING.md

```diff
@@ -223,18 +223,17 @@ Consult previous versions of this document for older versions of Node.js:
 
 ### Note about Python 2 and Python 3
 
-The Node.js project uses Python as part of its build process and has
+The Node.js project uses Python as part of its build process and was
 historically only been Python 2 compatible.
 
 Python 2 will reach its _end-of-life_ at the end of 2019 at which point the
 interpreter will cease receiving updates. See https://python3statement.org/
 for more information.
 
-The Node.js project is in the process of transitioning its Python code to
-Python 3 compatibility. Installing both versions of Python while building
-and testing Node.js allows developers and end users to test, benchmark,
-and debug Node.js running on both versions to ensure a smooth and complete
-transition before the year-end deadline.
+The Node.js project now supports both Python 3 and Python 2 for building.
```

Comment:
Could delete the "now", its obviously a statement about how it is now.

Reply:
```suggestion
The Node.js project supports both Python 3 and Python 2 for building.
```